site/layouts/shortcodes/features.html

```diff
@@ -6,7 +6,7 @@
     </tr>
   </thead>
 <tbody>
-{{- range $.Page.Pages }}
+{{- range sort $.Page.Pages "Params.jdkversion"  "desc"}}
```

Comment:
This is what it looks like when I build it myself: 
![image](https://github.com/marchof/java-almanac/assets/104227937/2365a830-f28b-41e8-8536-b94906938c25)